.github/workflows/frontend.yaml

```diff
@@ -48,3 +48,26 @@ jobs:
 
             - name: Check if the compilation is up to date
               run: git diff --no-patch --exit-code
+
+    test:
```

Comment:
This should be added to dev-kit with an if that checks if vitest is on package.json